R/pkg/R/functions.R

```diff
@@ -1008,6 +1008,24 @@ setMethod("round",
             column(jc)
           })
 
+#' bround
+#'
+#' Returns the value of the column `e` rounded to `scale` decimal places using HALF_EVEN rounding
+#' mode if `scale` >= 0 or at integral part when `scale` < 0.
+#'
+#' @rdname bround
+#' @name bround
+#' @family math_funcs
+#' @export
+#' @examples \dontrun{bround(df$c, 0)}
+setMethod("bround",
+          signature(x = "Column", scale = "numeric"),
```

Comment:
Oh, thank you for sharing know-how. I didn't know that how to do!
I will fix right now.